### ISSUE TITLE
Strange issues /w install and update

### ISSUE THREAD

**Issue by salsadoom** [FezVrasta/ark-server-tools#799]
Hi,

So I've installed arkmanager following the instructions, although on Archlinux instead of Ubuntu or Fedora. It seems to work fine, the servers are configured and working but I had a few issues and will fail again soon.. ok first issue.

"arkmanager install" did not actually do much, it made the directory structure ie, "~/ARK/ShooterGame/" but didn't actually put any files inside those directories. I couldn't find an error message in any logs that I know about. But I had a copy of the game downloaded earlier so I just put the files in place and it worked...

.. until an update came out, where it deleted the server binary. arkmanager.log and arkserver.log show no errors, running arkmanager update --verbose shows no errors, but doesn't download any files ever. Manually running "steamcmd +login anonymous +force_install_dir /home/$USER/ark/ +app_update 376030 validate +quit" does fix it temporarily, until the next update comes out.

Interestingly, installing or updating a mod actually works totally fine. 

I'm not sure where to look for solving this issue. My first thought was permissions but they seem to be fine. Besides, if there was a permissions issue it would effect installing and updating mods.

Any suggestions on where to look for the problem?

**Comment by klightspeed** [comment]
Do you have a staging directory set up?
If so, is there anything in that staging directory, and can the user write to it?

**Comment by salsadoom** [comment]
OK I wanted to do some testing to be sure, the problem was I had a copy of steamcmd installed from the package manager and it seemed to be running that copy, when I removed the distro copy and just used steamcmd in a directory as the instructions suggest it seems to be working fine. 

Before I did this, I did test with and without a staging directory. Seems odd it installed mods fine but its fixed now anyway. :)
